src/cmd_vel_mux.cpp

```diff
@@ -79,79 +104,109 @@ CmdVelMux::CmdVelMux(rclcpp::NodeOptions options) : rclcpp::Node("cmd_vel_mux",
   RCLCPP_DEBUG(get_logger(), "CmdVelMux : successfully initialized");
 }
 
-void CmdVelMux::configureFromParameters(const std::vector<std::string> & names, const std::vector<std::string> & topics, const std::vector<double> & timeouts, const std::vector<int64_t> & priorities, const std::vector<std::string> & short_descs)
+bool CmdVelMux::parametersAreValid(const std::map<std::string, ParameterValues> & parameters)
+{
+  for (const std::pair<std::string, ParameterValues> & parameter : parameters)
+  {
+    if (parameter.second.topic.empty())
+    {
+      RCLCPP_WARN(get_logger(), "Empty topic for '%s'", parameter.first.c_str());
+      return false;
+    }
+    else if (parameter.second.timeout < 0.0)
```

Comment:
@clalancette @Lobotuerk no need for `else` if the previous `if` clause does `return`.